Tests/Development.server.lua

```diff
@@ -7,7 +7,25 @@ MetrikSDK.Server:SetProjectId(ServerStorage.__Project_Id.Value)
 MetrikSDK.Server:SetAuthenticationToken(ServerStorage.__Project_Auth.Value)
 
 MetrikSDK.Server:InitializeAsync():andThen(function()
-	-- warn("Metrik SDK loaded!")
+	warn("Metrik SDK loaded!")
+	
+	local action = MetrikSDK.Server.ActionBuilder.new()
+		:SetName("Hello, world!")
+		:SetDescription("A simple 'Hello, World' print!")
+		:AddArgument("String", {
+			Type = "string",
+			Required = false,
+			Description = "Anything else you want to print?"
+		})
+		:Build()
+
+	function action:OnRun(source: string?)
```

Comment:
would it make sense to change this to an event of some sort and add `:Connect`? Doesn't that match existing Roblox patterns

Reply:
the Existing roblox pattern is mostly singletons - this is an object (for reference, you could take a look at a DataStorePages instance) and so we're probably not going to provide signals, but may be something to do later down the line if requested.

Reply:
ok